units/pod_creation.go

```diff
@@ -191,6 +215,18 @@ func (j *podCreationJob) populateIfUnset(ctx context.Context) error {
 	return nil
 }
 
+func (j *podCreationJob) waitForPodDefinition(family string) (*definition.PodDefinition, error) {
```

Comment:
I named it this way because the job is intended to poll repeatedly on each job attempt until the pod definition is eventually created by the pod definition job. I don't have that strong of feelings about this though, so I renamed it.

Reply:
Ok thanks!